service2/server.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const axios = require("axios");
+const app = express();
+
+const PORT = process.env.PORT || 3001;
+const SERVICE_PORT = process.env.SERVICE_ADDRESS || 3000;
```

Comment:
This isn't correct, the default address needs to be `http://localhost:3000` (as a string)
It's not the `SERVICE_PORT` it's still the `SERVICE_ADDRESS`